frontend/src/component/demo/demo-setup.ts

```diff
@@ -27,46 +62,10 @@ const ensureUserIdContextExists = async () => {
     }
 };
 
-export const specificUser = async () => {
-    await deleteOldStrategies('demoApp.step2');
-    await ensureUserIdContextExists();
-};
-
-export const gradualRollout = async () => {
-    await deleteOldStrategies('demoApp.step3');
-    const featureId = 'demoApp.step3';
-
-    const { environments }: IFeatureToggle = await fetch(
-        formatApiPath(
-            `api/admin/projects/${PROJECT}/features/${featureId}?variantEnvironments=true`,
-        ),
-    ).then((res) => res.json());
-
-    const strategies =
-        environments.find(({ name }) => name === ENVIRONMENT)?.strategies || [];
-
-    if (!strategies.find(({ name }) => name === 'flexibleRollout')) {
-        await fetch(
-            formatApiPath(
-                `api/admin/projects/${PROJECT}/features/${featureId}/environments/${ENVIRONMENT}/strategies`,
-            ),
-            {
-                method: 'POST',
-                headers: {
-                    'Content-Type': 'application/json',
-                },
-                body: JSON.stringify({
-                    name: 'flexibleRollout',
-                    constraints: [],
-                    parameters: {
-                        rollout: '50',
-                        stickiness: 'default',
-                        groupId: featureId,
-                    },
-                }),
-            },
-        );
-    }
+const cleanUpContext = async () => {
+    const contextFields = await getContextFields();
+    await deleteOldContextFields(contextFields);
```

Comment:
Mmmm this looks a bit odd... it looks like we're doing something via API that should be done via a migration...